podcasts/Kids Profile/KidsProfileBannerViewModel.swift

```diff
@@ -5,6 +5,7 @@ class KidsProfileBannerViewModel {
     var onRequestEarlyAccessTap: (() -> Void)? = nil
 
     func closeButtonTap() {
+        UserDefaults.standard.setValue(true, forKey: Constants.UserDefaults.kidsProfile.shouldHideBanner)
```

Comment:
We have the `Settings` for stuff like this. However, it's a singleton, so I understand moving away from it.

If we want to keep `UserDefaults.standard` maybe we can inject it?

Reply:
Thanks @leandroalonso ! I was looking for it!